doc/apps/dash.md

```diff
@@ -30,6 +30,11 @@ app = Dash(__name__)
 server = app.server
 ```
 
+```{note}
+Dash applications will run with Python 3.11. Refer to this [section](customize-deployment) for customized deployments.
+```
```

Comment:
no need to add a note, just mention in the first paragraph that the app runs on python 3.11

Reply:
[Fixed](https://github.com/ploomber/doc/pull/196/files#diff-232e0474ab7eb173493b1cbce28404de549efcd15343850ed95d1bfdd27720baR20-R22)